backend-spring-boot/src/main/java/com/pharmacy/cpis/util/config/WebSecurityConfig.java

```diff
@@ -69,6 +69,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .authorizeRequests()
                 .antMatchers("/auth/**").permitAll()
                 .antMatchers("/h2-console/**").permitAll()
+                .antMatchers("/api/consulltations").permitAll()
 
```

Comment:
If you are done with this PR, delete this. We do not want public access to this.

Reply:
Done